app/views/local_petitions/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<% content_for(:page_title) { "Petitions in #{@constituency.name}" } %>
+<h1>Petitions in <%= @constituency.name %></h1>
+
+<div class="section-panel local-petitions">
+  <% if @petitions.any? %>
+    <ol>
+      <% @petitions.each do |petition| %>
+        <li class="petition-item petition-<%= petition.state %>">
+          <%= link_to petition.title, petition_path(petition) %>
+          <p>
+            <%= number_with_delimiter petition.constituency_signature_count %> signatures from your constituency
+            (<%= number_with_delimiter petition.signature_count %> signatures total)
+          </p>
+        </li>
+      <% end -%>
+    </ol>
+  <% else %>
+    <p class="no_petitions_warning">No petitions are popular in your constituency.</p>
```

Comment:
nope, just copied it from elsewhere (possibly an old elsewhere that doesn't exist) - will remove.